src/styles/blocks/catalog.scss

```
@@ -18,8 +20,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: $during-anim;
+
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
   }
+
+  &__card:hover {
+    transform: scale(1.2);
+    color: #34568b;
+  }
```

write it inside previous block &__card {}
```suggestion
  &:hover {
    transform: scale(1.2);
    color: #34568b;
  }
```